lib/ldclient-rb/ldclient.rb

```diff
@@ -217,8 +218,13 @@ def initialized?
     # @return the variation for the provided context, or the default value if there's an error
     #
     def variation(key, context, default)
-      detail, _, _, = variation_with_flag(key, context, default)
-      detail.value
+      context = Impl::Context::make_context(context)
+      result = evaluate_with_hooks(key, context, default, :variation_detail) do
```

Comment:
In order for the correct method to reported in the hook wouldn't the symbol need to be `:variation`

Reply:
Good catch.